src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -236,14 +241,21 @@ class RunWorkflowTests : SeleniumTest()
     }
 
     @Test
-    fun `can select workflow progress tab and selecting a workflow option generates reports table`()
+    fun `can create a workflow and selecting view progress link navigates to progress tab and generates reports table`()
```

Comment:
Could perhaps add a test that the user can change the selected workflow on the progress page.

Reply:
That would be good- and also that selected workflow persists when you navigate away from the progress page and back again.